x/stakeibc/simulation/add_validator.go

```diff
@@ -19,7 +19,7 @@ func SimulateMsgAddValidator(
 	return func(r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context, accs []simtypes.Account, chainID string,
 	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
 		simAccount, _ := simtypes.RandomAcc(r, accs)
-		msg := &types.MsgAddValidator{
+		msg := &types.MsgAddValidators{
```

Comment:
do we actually run these sim tests? if not let's remove this

Reply:
Should we just wipe the whole folder? If so, I'll do that in a separate PR

Reply:
chatted offline - planning to remove in separate PR